README.md

```diff
@@ -92,6 +92,13 @@ When you want to spice things up a bit:
 $ timeout $RANDOM vim
 ```
 
+## The Shoot First, Ask Questions Later way
+Credit: @aliva
+ 
+```
+$ ps axuw | awk '{print $2}' | shuf | head -n 1 | sudo kill -9
```

Comment:
shuf support `-n 1` to only get the first entry. Also you should remove the `PID` entry. Better thefore:
```
ps axuw | awk '{print $2}' | greo -v PID | shuf  -n 1 | sudo kill -9
```

Reply:
good idea, thanks 👍